src/actions/contest_actions.js

```diff
@@ -14,6 +15,13 @@ export const receiveContests = payload => ({
 
 export const fetchContests = () => async dispatch => {
   const payload = await ContestAPIUtil.fetchContests();
+  Object.keys(payload.contests).forEach( contest_id => {
+    const winning_entry = {
+      entry: payload.contests[contest_id].winning_entry,
+      contest: payload.contests[contest_id],
+    };
```

Comment:
This would work, but let's stay consistent with our Redux state. The winning entry really should go in the entry slice of state, and the contest has the appropriate ID to pull it out?

Reply:
The winning entry does go to the entry slice of state - it's simply that the contests and entries reducers both have receive_entry at the moment, and are looking for objects with these keys - we can discuss? The most sensible solution to me seems like it would be taking receive_entry out of contestsreducer

Reply:
Ah, I see the issue. I tend to write the reducers in such a way that only one action needs to be dispatched from the async action. Therefore I would add a "RECEIVE_CONTESTS" case to the entries reducer that knows to grab all the winning entries from this payload. Does that make sense?

Reply:
Done and done!